pkg/didcomm/protocol/didexchange/states_test.go

```diff
@@ -678,7 +549,7 @@ func TestVerifySignature(t *testing.T) {
 		require.NoError(t, err)
 
 		cs := &ConnectionSignature{
-			Type:       "https://didcomm.org/signature/1.0/ed25519Sha512_single",
+			Type:       "https://didcomm.org;spec/signature/1.0/ed25519Sha512_single",
```

Comment:
there is no ";spec"

Reply:
yea there is no spec removed

Reply:
I still see it?

Reply:
just pushed the patch should not be there anymore